ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -180,9 +180,21 @@ peerChurnGovernor tracer deadlineChurnInterval bulkChurnInterval psOverallTimeou
       -- Give the promotion process time to start
       threadDelay 1
 
-      -- Forget the worst performing non-active peers.
+      -- Forget the worst performing established peers.
       atomically $ do
         decreaseEstablished churnMode ubp
+        modifyTVar peerSelectionVar (\targets -> targets {
+            targetNumberOfEstablishedBigLedgerPeers =
+              decrease (targetNumberOfEstablishedBigLedgerPeers base -
+                        targetNumberOfActiveBigLedgerPeers base)
+              + targetNumberOfActiveBigLedgerPeers base
+          })
```

Comment:
It will be more consistent if this is named as `decreaseEstablishedBigLedgerPeers`.

Reply:
👍